### PR TITLE
fix(amplify-codegen): excludes flutter projects from gql gen

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Makes sure that flutter project customers are not asked about generating GraphQL statements and
types.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-codegen/src/callbacks/prePushAddCallback.js [hunk 1]
```diff
@@ -16,11 +16,15 @@ async function prePushAddCallback(context, resourceName) {
 
   let shouldGenerateCode = false;
   if (context.exeInfo.inputParams) {
-    normalizeInputParams(context);
-    const inputParams = context.exeInfo.inputParams[constants.Label];
-    const yesFlag = context.exeInfo.inputParams.yes;
+    const frontend = getFrontEndHandler(context);
 
-    shouldGenerateCode = await determineValue(inputParams, yesFlag, 'generateCode', true, () => askShouldGenerateCode());
+    if (frontend !== 'flutter') {
```
Comment (akshbhu) [inline packages/amplify-codegen/src/callbacks/prePushAddCallback.js:21]: Is this skip also required when codegen is already setup in this line: 13?

**Review by akshbhu — DISMISSED** [review]
This PR evades gql Codegen during amplify push.
  
What about commands like ```amplify codegen statements```? Does this skip also required in such commands?

File: packages/amplify-codegen/src/callbacks/prePushAddCallback.js (hunk 1 above)
Comment (haverchuck) [reply packages/amplify-codegen/src/callbacks/prePushAddCallback.js:21]: To my knowledge the only way to configure codegen is to run the amplify codegen add command, which yields the following:
```
==> amplify-dev codegen add
flutter is not supported by codegen plugin
```

Therefor line 13 should never be executed in a flutter project, correct?

**Comment by haverchuck** [comment]
> This PR evades gql Codegen during amplify push.
> 
> What about commands like `amplify codegen statements`? Does this skip also required in such commands?

@akshbhu I believe commands such as `amplify codegen statements` first require `amplify codegen add`, which  as noted above fails gracefully for Flutter projects.  So we should be ok.

**Review by edwardfoyle — APPROVED** [review]

**Review by jhockett — APPROVED** [review]

**Pull request merged**